apiserver/facades/schema.json

```diff
@@ -22831,18 +22831,6 @@
                     },
                     "description": "InstanceTypes returns instance type information for the cloud and region\nin which the current model is deployed."
                 },
-                "ProvisioningScript": {
-                    "type": "object",
-                    "properties": {
-                        "Params": {
-                            "$ref": "#/definitions/ProvisioningScriptParams"
-                        },
-                        "Result": {
-                            "$ref": "#/definitions/ProvisioningScriptResult"
-                        }
-                    },
-                    "description": "ProvisioningScript returns a shell script that, when run,\nprovisions a machine agent on the machine executing the script."
-                },
```

Comment:
This file should never be affected with the changes we're doing.

Reply:
got it, thx